birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -333,6 +333,37 @@ for prov in ${WEAVER_WPS_PROVIDERS}; do
 done
 echo "${PREFIX}All Weaver remote WPS providers registered successfully!"
 
+if [ x"${WEAVER_UNREGISTER_DROPPED_PROVIDERS}" = x"True" ]; then
+    # Get all registered providers whether they are working or not
+    all_providers_resp=$( \
+        curl_cmd --insecure --silent --location \
+            -m ${REQUEST_TIMEOUT} \
+            -b "${cookie}" \
+            "${WEAVER_URL}/providers?check=false&detail=false" \
+    )
+    # Get all registered working providers
+    working_providers_resp=$( \
+        curl_cmd --insecure --silent --location \
+            -m ${REQUEST_TIMEOUT} \
+            -b "${cookie}" \
+            "${WEAVER_URL}/providers?check=true&detail=false" \
+    )
+    working_providers=$(echo "$working_providers_resp" | tr '\n' ' ' | \
+                        sed -e 's/.*"providers":\[//' -e 's/\].*//' -e 's/[",]/ /g')
+    for prov in $(echo "$all_providers_resp" | sed -e 's/.*"providers":\[//' -e 's/\].*//' -e 's/[",]/ /g'); do
+        # Note: both WEAVER_WPS_PROVIDERS and working_providers are whitespace delimited with no newlines
+        if echo " ${WEAVER_WPS_PROVIDERS} " | grep -qv "[[:space:]]${prov}[[:space:]]" && \
```

Comment:
This will be run each time autodeploy runs, which runs inside its own container.  Please test this with autodeploy.

The reason `curl` was a `docker run` is because there are no `curl` in the autodeploy container.

Reply:
can confirm this check works in the autodeploy container: `grep`, `sed`, etc. all work in that environment.